src/test/regress/expected/aggregate_support.out

```diff
@@ -561,9 +560,20 @@ create aggregate sumstring(text) (
 	combinefunc = sumstring_sfunc,
 	initcond = '0'
 );
-select sumstring(valf::text) from aggdata where valf is not null;
-ERROR:  function "aggregate_support.sumstring(text)" does not exist
-CONTEXT:  while executing command on localhost:xxxxx
+-- verify that the aggregate is propagated
```

Comment:
Can you please add more aggregates to test
- Having non-distributable dependency type on aggregate
- Having undistributed table dependency on aggregate
- With various samples from https://www.postgresql.org/docs/14/sql-createaggregate.html (Including samples from https://www.postgresql.org/docs/14/xaggr.html would be enough)
- With old syntax
- Anything else you come up with